lib/travis/cli/api_command.rb

```diff
@@ -108,7 +108,7 @@ def setup_enterprise
             endpoint
           end
           self.api_endpoint             = c[enterprise_name]
-          self.insecure                 = true if insecure.nil?
+          self.insecure                 = true unless insecure.nil?
```

Comment:
I was trying to keep consistent with that against `api.travis-ci.org`. will improve it with a test